connect/2.1/Dockerfile.local

```diff
@@ -15,6 +15,8 @@ COPY debezium-connector-oracle-$DEBEZIUM_VERSION-SNAPSHOT-plugin.tar.gz $KAFKA_C
 COPY debezium-connector-db2-$DEBEZIUM_VERSION-SNAPSHOT-plugin.tar.gz $KAFKA_CONNECT_PLUGINS_DIR/debezium-db2-plugin.tar.gz
 COPY debezium-connector-vitess-$DEBEZIUM_VERSION-SNAPSHOT-plugin.tar.gz $KAFKA_CONNECT_PLUGINS_DIR/debezium-vitess-plugin.tar.gz
 
+ADD debezium-connect-rest-extension-$DEBEZIUM_VERSION-SNAPSHOT.tar.gz $EXTERNAL_LIBS_DIR
```

Comment:
I think the correct extension here is ``jar`` as running the assembly profile locally doesn't seem to produce any tar archives for debezium-connect-rest-extension module.

Reply:
Yes, great catch. I didn't realize there wasn't a .tar.gz file for this artifact. I have a change locally for this for the Maven download as well. I'm working on Jiri's request to commonize some logic and will have those changes up shortly. I was running into problems testing fully with buildx yesterday because of self-signed certificates, but I've got that working now as well so doing more testing as well.

```
docker-maven-download central io/debezium debezium-connect-rest-extension "$DEBEZIUM_VERSION" "$KCRESTEXT_MD5" && \
```

Reply:
@dude0001  what issue with buildx? I can't think of anything where certificates should be an issue. I'm currently running the build for arm to verify if the failure in GH actions was a temporary glitch or if there is some issue with npm installation again. 

If you can fix the extension I will verify the full functionality as I have the change to release pipeline ready.

Reply:
It is an issue in my personal development I was able to work through. We have some software that inspects all SSL traffic on the network that many CLI tools see rightly as a man-in-the-middle attack. I have to hack the dockerfiles in my local development environment so Docker, the OS in the Docker containers and CLI tools running in Docker containers trust the self-signed certs on my network used for this purpose. The product is Netskope but there are a few other tools that security organizations deploy internally to enterprises these days that is a nightmare for developers. I usually am able to work around it fine but was having an issue with buildx itself and how it pulls images from registries, but finally figured it out this morning and was able to reconfigure my local environment for it. No issue with Debezium or this repo.

Reply:
I've changed everything to assume a .tar.gz artifact will be available. For local testing, I hacked something to simulate that and I think it will work once that artifact is available. Please have another look.